### DISCUSSION TITLE
project license access

### DISCUSSION THREAD

**Discussion by ITJamie** [python-gitlab/python-gitlab#2611] [Q&A]
## Description of the problem, including code/CLI snippet

I currently cannot see a method to access the license data of a project.
there is an extension in the gitlab API to return the license of a project (eg https://gitlab.com/api/v4/projects/19352197?license=true) which will add the "license" block to the project endpoint response.

## Expected Behavior

ability to access the license on a project

## Actual Behavior

no ability (or docs) to access the license block on a gitlab project page

## Specifications

  - python-gitlab version: 3.15.0
  - API version you are using (v3/v4): v4
  - Gitlab server version (or gitlab.com): gitlab.com

**Comment by ITJamie** [comment]
if anyone needs this for future, i was able to workaround this with:
```python
project_details = gl.http_get(path= f"/projects/{project_id}?license=True")
print(project_details['license'])
```

**Comment by JohnVillalovos** [comment]
Probably can do:

```
gl = gitlab.GitLab(...)
project = gl.projects.get(12345, license=True)
```

Or something like that.

Additional arguments can be specified in the call to `get()`

See: https://python-gitlab.readthedocs.io/en/main/api-usage.html#managers

Section in there that contains the text "python-gitlab allows to send any data to the GitLab server when making queries."
